.pep8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 100
```

Comment:
contradicts `max_line_length=119` in `.editorconfig`

Reply:
true :)
i missed that. Editorconfig used for cross IDE setup, pep8 file for flake setup...